chart/templates/rbac.yaml

```diff
@@ -89,6 +89,59 @@ rules:
     - list
     - update
     - patch
+- apiGroups:
+  - constraints.gatekeeper.sh
```

Comment:
can we have some conditions if related API available, like

```
{{- if .Capabilities.APIVersions.Has "constraints.gatekeeper.sh" }}
```

Reply:
Hi @eumel8 I think it could be problematic in some specific use cases

1) Installation made without Helm (helm template and then kubectl over the file extracted from the helm template)
2) Installation together with Gatekeeper (helm chart with both inside, even though I think you can give a particular order precedence)

Example for the first use case:

- Company A & B: installs different charts using helm (Istio, VPA, Karpenter, Kube-Downscaler, ...) -> OK
- Company C: creates its helm chart collecting each template needed (helm template istio, helm template vpa, helm template karpenter, helm template gateekeeper). If I'm understing correctly the Company couldn't create its helm template for the KubeDownscaler if Gatekeeper CRDs are not installed inside the Cluster they are using together with helm to extract the templates -> Could be problematic